content/docs/guides/branch-archiving.md

```diff
@@ -28,11 +28,13 @@ To minimize storage costs, Neon automatically archives branches that are:
 
 Both conditions must be true for a branch to be archived.
 
-Additionally, these conditions apply:
+However, a branch **cannot** be archived if it:
 
-- A branch cannot be archived if it has an unarchived child branch.
-- A child branch must be archived before a parent branch can be archived.
-- [Protected branches](/docs/guides/protected-branches) are not archived.
+- Has an **unarchived child branch**.
+- Has **computes running**.
+- Is **in transition** (e.g., currently being created or unarchived).
+- Is a **protected branch** ([learn more](/docs/guides/protected-branches)).
+- Was created **less than 10 minutes ago** and has undergone a state change **within the last 5 minutes** (to prevent rapid archive/unarchive cycles).
```

Comment:
this should be **more than 10 minutes ago**. Because branches might be archived right after creation (for instance if it's a some unused backup branch just created). So, less than 10 minutes ago and a state change in 5 minutes is actually okay -- this is useful for some use cases. However, if the branch is 10 minutes old then it's "old enough" so we forbid state changes if there was one in 5 minutes already.

Reply:
Also, note that this condition applied so automatic archival in response to limit hit. "Normal" archival does not check this because it has TTL and threshold. So, maybe we could make this more generic: "There is some protection from cycles". because I hope that normally no one hits this condition

Reply:
I'll take it out entirely. It's a bit complex. If we find that it's an issue, we can add it back.